compilers.md

```diff
@@ -41,16 +41,70 @@ based on the NVIDIA/PGI commercial compiler.
 ### Intel
 
 [Intel Parallel Studio XE](https://software.intel.com/en-us/parallel-studio-xe)
-is Intel's suite of compilers, tools, and libraries for Fortran, C, C++, and 
-Python. 
-Intel offers the Linux edition of their compiler suite for free for
+is Intel's suite of compilers, tools, and libraries for Fortran, C, C++, and
+Python. Intel offers the Linux edition of their compiler suite for free for
 [open source contributors](https://software.intel.com/en-us/parallel-studio-xe/choose-download/open-source-contributor).
 
-### IBM
+### NAG
 
-[IBM XL Fortran Compiler](https://www.ibm.com/us-en/marketplace/xl-fortran-linux-compiler-power) is available for Linux on POWER9 architectures.
+The latest [NAG Fortran Compiler](https://www.nag.com/nag-compiler)
+release (7.0) has extensive support for legacy and modern Fortran features including parallel programming with coarrays, as well as additional support for programming with OpenMP.
 
-### NAG
+The Compiler also provides significant support for Fortran 2018 (atomic
+operations, events and tasks, plus other smaller features), almost all of
```

Comment:
What are tasks? Do you mean teams?

Reply:
I just copied what was at their page.

Reply:
I would suggest to just keep it. It's exactly their wording, so our job should be to just update it if they change their website, or they tell us what wording to use. Otherwise we should assume their website is correct.

Reply:
Sounds good. Could be a NAG extension.